README.md

```diff
@@ -1,227 +1,47 @@
-# Starter kit for a Terraform GitOps module
+# Cloud Pak for Data, db2oltp Subscription and Db2oltpService instance gitops module
```

Comment:
We should probably add a note here saying that the unit test is expected to be executed on a cluster that already has CP4D-instance and its dependencies installed and configured.